specs/proposals.md

```diff
@@ -124,7 +124,7 @@ function deleteL2Output(bytes32 _l2Output) external
 /**
```

Comment:
We will probably need to go through the specs and update them more but that is fine to do in the future